[chore] Re-record JSON responses for docs

nwithan8 opened the pull request:
- Most JSON responses have not changed outside of timestamps. 
- Some have extra data entries (previously one or two records, now three or four).
- Some deprecated functionality (carbon offset, batch create+buy) have been removed.
- Added Python tool used to generate these response snippets + improved documentation and Makefile steps

Justintime50 reviewed (dismissed):
I'd like these regenerated using the correct user ID for the responses please, could be some weird artifacts mixed in if not (see internal diff for more details on that)

nwithan8 commented:
> I'd like these regenerated using the correct user ID for the responses please, could be some weird artifacts mixed in if not (see internal diff for more details on that)

I have addressed your feedback and re-recorded all the cassettes with the proper API keys, did result in a lot less diffs.

I've also added the Python tool we use to generate these snippets into the `tools` directory and improved its documentation so it will be easier next time around to carry out this process.

Justintime50 commented on official/docs/responses/addresses/addresses-create-and-verify.json:
```diff
@@ -22,26 +22,12 @@
     "verifications": {
       "zip4": {
         "success": true,
-        "errors": [
```
thought: we probably want to change the address here to one that will error. The entire point of this example was to show this in the response

Justintime50 commented on official/docs/responses/addresses/addresses-verify.json:
```diff
@@ -22,26 +22,12 @@
     "verifications": {
       "zip4": {
         "success": true,
-        "errors": [
```
thought: we probably want to change the address here to one that will error. The entire point of this example was to show this in the response

Justintime50 commented on official/tools/build_doc_json_responses/README.md:
```diff
@@ -0,0 +1,47 @@
+
+**NOTE**: Some static data used by this tool, such as the USPS carrier account ID, is associated with the EasyPost Docs Tooling User. You must use that user's API keys to generate these snippets.
+
+Generate response snippets are stored in the `responses` directory. Copy this entire directory to `official/docs/responses` once complete.
```
```suggestion
Generated response snippets are stored in the `responses` directory. Copy this entire directory to `official/docs/responses` once complete.
```

Justintime50 commented on official/tools/build_doc_json_responses/README.md:
```diff
@@ -0,0 +1,47 @@
+
+## Troubleshooting
+
+### A bunch of "F"s
```
thought: this is a redefinition of another paragraph above, seems odd to repeat it said slightly differently. Can we consolidate them?
